src/SprykerEco/Zed/UnzerApi/Business/Api/Request/SetWebhookUrlRequest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request;
+
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use Symfony\Component\HttpFoundation\Request;
+
+class SetWebhookUrlRequest extends UnzerApiAbstractRequest implements UnzerApiRequestInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return string
+     */
+    public function getUrl(UnzerApiRequestTransfer $unzerApiRequestTransfer): string
+    {
+        return $this->unzerApiConfig->getUnzerApiSetWebhookUrl();
+    }
+
+    /**
+     * @return string
+     */
+    public function getHttpMethod(): string
+    {
+        return Request::METHOD_POST;
+    }
```

Comment:
This method is really often in use ... something like e.g. UnzerApiAbstractPostRequest does make sense?

Reply:
Fixed